### PR TITLE
Python program to find LCM of an array

### PR DESCRIPTION
**Pull request by himi98**
### Describe your change:
Added an algorithm to find the LCM of array elements.

* [X] Add an algorithm?
* [X] Fix a bug or typo in an existing algorithm?
* [X] Documentation change?

### Checklist:
* [X] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [X] This pull request is all my own work -- I have not plagiarized.
* [X] I know that pull requests will not be merged if they fail the automated tests.
* [X] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [X] All new Python files are placed inside an existing directory.
* [X] All filenames are in all lowercase characters with no spaces or dashes.
* [X] All functions and variable names follow Python naming conventions.
* [X] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [X] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [X] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [X] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: strings/lcm_of_array.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+def gcd(a, b):
+    if isprime(a) and isprime(b):
+        return 1
+    elif b == 0:
+        return a
+    else:
+        return gcd(b, a % b)
+
+
+n = int(input())
```
Comment (ChrisO345) [inline strings/lcm_of_array.py:20]:
```suggestion
```
This never gets used

File: strings/lcm_of_array.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+lst = list(map(int, input().split()))
+lcm = lst[0]
+for i in range(1, len(lst)):
+    # print(i)
+    lcm = (lst[i] * lcm) // (gcd(lst[i], lcm))
+    # print(gcd(lst[i],lcm))
+print(lcm)
```
Comment (ChrisO345) [inline strings/lcm_of_array.py:27]: Can you put all of this into a function and include some doctests: https://docs.python.org/3/library/doctest.html

**Review by ChrisO345 — CHANGES_REQUESTED** [review]

**Comment by ChrisO345** [comment]
This doesn't seem to be working, if given inputs `[10, 15, 20, 60]` the result should be 60 not 300. It also doesn't work for smaller arrays such as `[10, 15]` which should be 30 not 150.

**Comment by cclauss** [comment]
[CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md) was not read.  Please do not open new pull requests in this repo until November 1st. 2022.

**Pull request closed without merging**